go_lib/deckhouse-config/validator.go

```diff
@@ -93,30 +77,28 @@ func (c *ConfigValidator) ValidateCR(cfg *d8cfg_v1alpha1.ModuleConfig) Validatio
 	// Can run conversions and validations if spec.version and spec.settings are specified.
 	if cfg.Spec.Settings == nil {
 		// Warn about spec.version without spec.settings.
-		result.ValidateCRWarning = "spec.version is redundant without spec.settings"
-	} else {
-		// Resource has both spec.settings and spec.version — it is ok to convert and validate with OpenAPI schema.
-		result.HasVersionedSettings = true
+		result.Warning = "spec.version is redundant without spec.settings"
```

Comment:
"spec.version has no effect without spec.settings, defaults from the latest config version will be applied"